smdebug/pytorch/hook.py

```diff
@@ -69,6 +70,11 @@ def _get_num_workers(self):
                     return hvd.size()
             except (ModuleNotFoundError, ValueError, ImportError):
                 pass
+
+        if not is_first_process(self.out_dir):
+            self.save_all_workers = False
```

Comment:
what happens when include_workers="all"?

Reply:
The `include_workers=all` flag have no bearing since it is only relevant for supported distributed training techniques, the `is_first_process` is used to detect unsupported distributed techniques.